src/shogun/distributions/Gaussian.cpp

```diff
@@ -404,9 +405,10 @@ SGVector<float64_t> CGaussian::sample()
 	}
 
 	SGVector<float64_t> random_vec(m_mean.vlen);
+	NormalDistribution<float64_t> normal_dist;
```

Comment:
@vigsterkr @theartful question: wouldn't it make sense to pass the rng in the ctor argument?

Reply:
@karlnapf as mentioned earlier check standard...

Reply:
ok thx

Reply:
@karlnapf we're following the way the standard library does it. pass prng to operator()